python/ccf/clients.py

```diff
@@ -537,24 +553,30 @@ def _response(self, response: Response) -> Response:
         LOG.info(response)
         return response
 
-    def _direct_call(
+    def _call(
         self,
         path: str,
         body: Optional[Union[str, dict, bytes]] = None,
         http_verb: str = "POST",
         headers: Optional[dict] = None,
         signed: bool = False,
         timeout: int = DEFAULT_REQUEST_TIMEOUT_SEC,
+        log_capture: Optional[list] = None,
     ) -> Response:
         description = ""
         if self.description:
-            description = f"({self.description})" + (" [signed]" if signed else "")
+            description = f"{self.description}" + (" (sig)" if signed else "")
```

Comment:
I think `signed` is slightly more precise than `sig`?

Reply:
Yes, but it's also longer. It takes precious line space away from display more interesting stuff like the message itself so I'd like to reduce it as much as possible. I was thinking something like `[0|u0]s ...` would be nice (to node 0, as user 0, signed request).